### PR TITLE
Webhook type casting

### PR DESCRIPTION
**Pull request by vkresch**
---
name: Webhook type casting
about: Fix found user bug.
labels: 

---
Resolves https://github.com/tezos-reward-distributor-organization/tezos-reward-distributor/issues/417

* **Analysis**: No type casting is performed hence creating type issues for users.

* **Solution**: Perform explicit type casting of the webhook payload.
* **Implementation**: Added a type cast function.

* **Performed tests**: Added regression test with ints, floats and strings.

* **Documentation**: Added comments to the regression test.

* **Check list**:

- [x] I extended the Github Actions CI test units with the corresponding tests for this new feature (if needed).
- [ ] I extended the Sphinx documentation (if needed).
- [ ] I extended the help section (if needed).
- [ ] I changed the README file (if needed).
- [ ] I created a new issue if there is further work left to be done (if needed).

**Work effort**: 2h

### CODE REVIEW

File: tests/regression/test_webhook_send_admin_notification.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+rewardlog2.current_balance = "1000000000"
+rewardlog3.amount = "12345678900"
+rewardlog3.service_fee_amount = "5555555"
+rewardlog3.ratio = "0.16080255"
+rewardlog3.service_fee_ratio = "0.01398283"
+rewardlog3.service_fee_rate = "0.08"
+
+cfg = {
+    "webhook": {
+        "endpoint": "https://testnet-tezos.giganode.io:443",
```
Comment (utdrmac) [inline tests/regression/test_webhook_send_admin_notification.py:45]: This is not correct. Tezos RPC's do not support webhook. You need to change this to the original testing server.

Comment (vkresch) [reply tests/regression/test_webhook_send_admin_notification.py]: done

Comment (utdrmac) [reply tests/regression/test_webhook_send_admin_notification.py]: will that actually work? mydomain.com/webhook.php ?

Comment (vkresch) [reply tests/regression/test_webhook_send_admin_notification.py]: It's only a dummy configuration which does no actual requests. 
The test only tests if the payload was generated correctly with the correct types.

**Review by utdrmac — APPROVED** [review]

**Pull request merged**